semgrep-core/src/core/ast/AST_generic.ml

```diff
@@ -592,7 +594,6 @@ and literal =
                   escaped  unescaped
   *)
   | String of string wrap
-  | Regexp of string wrap bracket (* // *) * string wrap option (* modifiers *)
```

Comment:
Won't this make const-prop to stop working as we discussed in Slack?

Reply:
Yes. To avoid this problem, I restored this line and we now have separate expression node just for regexp templates.